### PR TITLE
feat(releases): Update docs to use GitHub App instead of service account PAT

### PR DESCRIPTION
**Pull request by Jeffreyhung**
## DESCRIBE YOUR PR

We are planning to replace the current usage of getsentry-release bot with a GitHub App, this PR will update the docs to include updated code template.

More details: https://www.notion.so/sentry/DACI-Replace-GitHub-bot-accounts-with-GitHub-Apps-getsentry-release-15109965d1204a91b9be71c49e8b66e0?pvs=4

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: develop-docs/sdk/processes/releases.mdx
Comment (coolguyzone) [inline develop-docs/sdk/processes/releases.mdx:147]: This url s 404ing for me, is it correct?

Comment (coolguyzone) [reply develop-docs/sdk/processes/releases.mdx:147]: Ah, I see the issue here, let's change it to a relative url:

```suggestion
Download and save the [default ruleset template](/json/Default_ruleset.json) as a JSON file.
```

**Review by coolguyzone — APPROVED** [review]
Looks good to me, I added a suggestion to reformat one of the links.

File: develop-docs/sdk/processes/releases.mdx
Comment (lizokm) [inline develop-docs/sdk/processes/releases.mdx:149]:
```suggestion
Visit the ruleset setting page of your repository: `https://github.com/getsentry/REPONAME_HERE/settings/rules`, click on the green **New ruleset** button, choose **Import a ruleset**, and select the JSON file you just downloaded. You can tweak the ruleset settings, but please don't remove the App in the Bypass List.
```

**Review by lizokm — APPROVED** [review]
LGTM

File: develop-docs/sdk/processes/releases.mdx
Comment (Jeffreyhung) [reply develop-docs/sdk/processes/releases.mdx:147]: @coolguyzone the changes I'm making to are on develop.sentry.dev, and the JSON is hosted on docs.sentry.io, hence I don't think the relative URL will work.
I would love to move the JSON over under the develop domain, but I'm not sure how to do that, any suggestions?

**Comment by Jeffreyhung** [comment]
Will be waiting for approval on the devinfra side on this before rolling this out

File: develop-docs/sdk/processes/releases.mdx
Comment (coolguyzone) [reply develop-docs/sdk/processes/releases.mdx:147]: Ah, fair point. In that case, the absolute URL makes sense.

Comment (Jeffreyhung) [reply develop-docs/sdk/processes/releases.mdx]: I was having incorrect assumption here, I thought the JSON folder is only available under docs.sentry.io, but it's also available for develop.sentry.dev. 
Updated link to make it a relative URL

**Comment by Jeffreyhung** [comment]
Got a green light on this! Merging!

**Pull request merged**
